app/routes/admin/api.py

```diff
@@ -0,0 +1,46 @@
+"""
+.. module:: api
+    :synopsis: All routes on the ``api`` Blueprint.
+
+.. moduleauthor:: Jett Andersen <jettca1@gmail.com>
+"""
+
+from datetime import datetime, timedelta, date
+
+from flask import Blueprint, request, render_template, g, redirect, \
+    url_for, flash, jsonify
+
+from bson.objectid import ObjectId
+from mongoengine.errors import DoesNotExist, ValidationError
+
+from bson import json_util
+import json
+
+from app.models import Event, Image
+from app.forms import CreateEventForm, EditEventForm, DeleteEventForm, UploadImageForm
```

Comment:
a lot of these imports don't seem to be necessary unless I'm confused as to how they're used